src/config/types.ts

```diff
@@ -249,6 +249,21 @@ type ReadinessProbeObj = {
 
 type ReadinessProbe = ReadinessProbeFn | ReadinessProbeObj;
 
+export enum RecordMode {
+    // Record and save all test runs
+    On = "on",
+    // Do not record any test runs
+    Off = "off",
+    // Record and save all retries
+    OnForRetries = "on-for-retries",
```

Comment:
`onRetries`?

Reply:
After some discussion we've agreed to have these options: on, off, retries-only, last-failed-run.